api/application.go

```diff
@@ -217,11 +217,11 @@ func (h ApplicationHandler) BucketGet(ctx *gin.Context) {
 
 // BucketUpload godoc
 // @summary Upload bucket content by ID and path.
-// @description Upload bucket content by ID and path.
+// @description Upload bucket content by ID and path (handles both [post] and [put] requests).
```

Comment:
Swagger rejected a method handling two methods, removed the put and added it just into the description.